notifications/providers/discord.py

```diff
@@ -20,13 +20,15 @@ def __init__(self):
                 self.config = json.load(json_file)
                 if self.config["webhook_url"]:
                     self.webhook_url = self.config["webhook_url"]
+                    self.user_id = self.config.get("user_id", "N/A")
```

Comment:
can this just be 
```
self.config.get("user_id")
```
then self.user_id would be None if its left empty.

Reply:
I use .isdigit() to check if the user has entered a user id.
Leaving self.user_id as None would throw an error.

Reply:
Ah ok.